### DISCUSSION TITLE
OTA Update

### DISCUSSION THREAD

**Discussion by johnbhansen** [jomjol/AI-on-the-edge-device#2931] [General] [answered]
When trying to "OTA Update" with v15.7.0 (or v15.6.0) I get the error message "File size must be less than 8MB!" 
My System, Info says:
--------------------------
SD card name: | SB32G
SD card size: | 30436 MB
SD card sector size: | 512 byte
Partition size: | 30424 MB
Partition free space: | 30405 MB
-------------------------
Total Free (Int + Ext): | 846119  byte
Ext. RAM - Free: | 750248  byte
Ext. RAM - Largest Free Block: | 737280  byte
Ext. RAM - Min Free: | 626440  byte
Int. RAM - Free: | 95871  byte
Int. RAM - Largest Free Block: | 90112  byte
Int. RAM - Min Free: | 89027 byte

Something I misunderstood...?

**Comment by caco3** [comment, marked as answer]
you seem to have selected a wrong file

**Reply by johnbhansen** [reply]
Thank you SO much !!!
